parsec/core/fs/userfs/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+from parsec.core.fs.userfs.userfs import UserFS
+
+
+__all__ = ("UserFS",)
```

Comment:
I'm not a big fan of having the user stuff being called `FS` when it doesn't have much to do with the file system logic. It feels like legacy, I suppose it's because the main user interface used to be called `FS`. I would keep `FS` as a workspace concept only.

Reply:
I agree, any idea of a name that could replace FS ?

Reply:
I couldn't find anything satisfying when I wrote my comment :sweat_smile: This class is mostly about administrating the workspaces right? Can we come up with a name that conveys this idea?

Reply:
> This class is mostly about administrating the workspaces right? Can we come up with a name that conveys this idea?

I fear we end up with som "admin" stuff (and we already have admin role and administration login with backend, so already too much of this...)

Reply:
Right, so no `Administrator/Manager`. What about `Hub`?